package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-camel-project",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
why do we have a diff here? Just because it would need to rebase on master?

Reply:
would you prefer that we do version changes in separate PRs? I honestly just thought that adding the initial cut of wsdl2rest support would also include a version bump

Reply:
I was expecting that the master branch was already updated for the "following version".
Fine to do it in this PR